docs/index.rst

```diff
@@ -54,6 +54,15 @@ decided to open source our work in hopes of helping the wider Network Defense co
 Since then, we've been constantly enhancing stoQ thanks to the feedback and contributions
 from the community of stoQ users.
 
+What is new in stoQ v3?
+***********************
+
+The recent changes made to stoQ were to meant for implementing a foundation for running and
+designing coroutines. The changes to date for stoQ v3 consist of:
+    - asyncio package
+    - Removal of DeepDispatcher functionality within the Architecture
```

Comment:
@mlaferrera I made a comment on the commit diff.  Basically, will there be something that replaces deep dispatching, in particular being able to make dispatching decisions based on scan results and being able to dispatch more than once for a given payload.

Reply:
@ytreister -- Absolutely. The plan right now is to add a new class `RequestState` (or something similarly named) which will be accessible to worker and dispatcher plugins. We believe this is a more streamlined and simpler solution when compared to `DeepDispatcher`. Please check out the v3 projects page if you'd like to keep up with the development process. Also, please feel free to create new issues if you think there are other feature requests that would be useful in v3.